dev_tools/install_macos.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ "$TRAVIS_OS_NAME" == "osx" ]; then
```

Comment:
If this script is specific to the continuous integration, give it a more descriptive filename so that users do not execute it by accident.